src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
```

Comment:
re: get more info -- done

re: localStorage stuff -- Since there is no programmatic way to do this, I think it would be better done in documentation. Would you like to do it? I think you are more familiar with it than I am.